### PR TITLE
add pages for legal policies and info

### PR DESCRIPTION
**Pull request by eshaben**
Add placeholder pages for the legal info

goes with: https://github.com/papermoonio/polkadot-mkdocs/pull/54

### CODE REVIEW

**Review by 0xLucca — APPROVED** [review]
LGTM!

**Review by albertov19 — APPROVED** [review]
LGTM

File: cookie-policy.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Cookie Policy
+description: This cookie policy explains how we use cookies to enhance your browsing experience, improve functionality, and ensure secure interactions.
+hide:
+- toc
+- feedback
+- navigation
+---
+
+--8<-- 'https://raw.githubusercontent.com/papermoonio/polkadot-mkdocs/refs/heads/main/legal/policies/cookie-policy.md'
```
Comment (albertov19) [inline cookie-policy.md:10]:
```suggestion
--8<-- 'https://raw.githubusercontent.com/papermoonio/polkadot-mkdocs/refs/heads/main/policies/cookie-policy.md'
```

File: privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Privacy Policy
+description: This privacy policy explains in particular how, for which purposes, and to what extent your Personal Data is collected and processed by us through the Website.
+hide:
+- toc
+- feedback
+- navigation
+---
+
+--8<-- 'https://raw.githubusercontent.com/papermoonio/polkadot-mkdocs/refs/heads/main/legal/policies/privacy-policy.md'
```
Comment (albertov19) [inline privacy-policy.md:10]:
```suggestion
--8<-- 'https://raw.githubusercontent.com/papermoonio/polkadot-mkdocs/refs/heads/main/policies/privacy-policy.md'
```

File: terms-of-use.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: Terms of Use
+description: Discover the terms and conditions governing your use of our website, outlining your rights, responsibilities, and important legal details.
+hide:
+- toc
+- feedback
+- navigation
+---
+
+--8<-- 'https://raw.githubusercontent.com/papermoonio/polkadot-mkdocs/refs/heads/main/legal/policies/terms-of-use.md'
```
Comment (albertov19) [inline terms-of-use.md:10]:
```suggestion
--8<-- 'https://raw.githubusercontent.com/papermoonio/polkadot-mkdocs/refs/heads/main/policies/terms-of-use.md'
```

**Review by albertov19 — CHANGES_REQUESTED** [review]
Fixed the URLs now that it is merged in mkdocs

**Review by 0xLucca — APPROVED** [review]

**Review by albertov19 — APPROVED** [review]

**Pull request merged**